msprime/species_trees.py

```diff
@@ -27,24 +27,65 @@
 
 def parse_starbeast(tree, generation_time, branch_length_units="myr"):
     """
-    Method to parse species trees produced by the program TreeAnnotator
-    based on posterior tree distributions generated with StarBEAST
-    (https://academic.oup.com/mbe/article/34/8/2101/3738283).
-    Species trees produced by TreeAnnotator are written in Nexus
-    (https://en.wikipedia.org/wiki/Nexus_file) format.
-
-    Species trees written by TreeAnnotator are rooted, bi-furcating, and
-    ultrametric. Branch lengths usually are in units of millions
-    of years ("myr"), but the use of other units is permitted by
-    StarBEAST (and thus TreeAnnotator). Here, we allow only units of
-    millions of years ("myr") or years ("yr"). Leafs must be named and
-    the tree must include information on population sizes of leaf and
-    ancestral species in the form of annotation with the "dmv" tag,
-    which is the case for trees written by StarBEAST and TreeAnnotator.
-    However, to convert these "dmv" values into absolute population
-    sizes, a generation time is required and must be provided separately
-    to the tree.
+    This function parses a species tree produced by the program `TreeAnnotator
+    <https://www.beast2.org/treeannotator>`_
+    based on a posterior tree distribution generated with `StarBEAST
+    <https://academic.oup.com/mbe/article/34/8/2101/3738283>`_  and defines a
+    simulation model according to the species tree. Species trees produced by
+    TreeAnnotator are written in `Nexus
+    <https://en.wikipedia.org/wiki/Nexus_file>`_ format and are rooted,
+    bifurcating, and ultrametric. Branch lengths usually are in units of millions
+    of years, but the use of other units is permitted by StarBEAST (and thus
+    TreeAnnotator). This function allows branch length units of millions of years
+    or years. Leafs must be named and the tree must include information on
+    population sizes of leaf and ancestral species in the form of annotation with
+    the "dmv" tag, which is the case for trees written by TreeAnnotator based on
+    StarBEAST posterior tree distributions.
+
+    After reading the input tree, this function defines a population configuration
```

Comment:
Be nice to cross link with the docs, e.g.  ``this function defines a :class:`PopulationConfiguration` for each...``. Similarly for the MassMigration below.

Reply:
Ah, I see you've done this below in the params docs. Feel free to ignore then, as you prefer.